### ISSUE TITLE
Add "ui5 versions" command to list versions of all UI5 Tooling modules

### ISSUE THREAD

**Issue by RandomByte** [SAP/ui5-cli#64]
## Expected Behavior
```sh
$ ui5 versions
@ui5/cli:		0.2.1
@ui5/builder:	0.2.3
@ui5/server:	0.2.0
@ui5/fs:		0.2.1
@ui5/project:	0.2.2
@ui5/logger:	0.2.0
```

## Current Behavior
Currently, only the @ui5/cli version can be retrieved via `ui5 --version`

## Affected components
- [ ] [ui5-builder](https://github.com/SAP/ui5-builder)
- [ ] [ui5-server](https://github.com/SAP/ui5-server)
- [x] [ui5-cli](https://github.com/SAP/ui5-cli)
- [ ] [ui5-fs](https://github.com/SAP/ui5-fs)
- [ ] [ui5-project](https://github.com/SAP/ui5-project)
- [ ] [ui5-logger](https://github.com/SAP/ui5-logger)

**Comment by RandomByte** [comment]
Hey @zhaoshengjun, thank you for your interest in submitting a PR to the UI5 CLI!

We actually just discussed this earlier this week and think the command should be named "ui5 versions" rather than "ui5 version" as it returns multiple versions. I updated the issue with the new name.

Regarding your question, the command should **exclusively** show the versions of the these modules:
- `@ui5/cli` (the project itself)
- `@ui5/builder`
- `@ui5/server`
- `@ui5/fs`
- `@ui5/project`
- `@ui5/logger`

The location should be the one those modules would be used from at runtime. If the CLI is installed globally, those modules should be taken from that global installation. If the CLI is installed locally within a project, those modules will be taken from there.

I think the easiest way might be to use the node `require()` function to read the package.json files of those dependencies. Something like this:
```
require("@ui5/fs/package.json")
```
